packages/openmrs/src/http.js

```diff
@@ -56,22 +57,37 @@ export function request(method, path, options = {}) {
  *      limit: 1
  *    }
  *  }
- *  )
+ * )
+ * @example <caption>Don't throw if OpenMRS returns a 404 error code</caption>
+ * http.get(
+ *  "/ws/rest/v1/patient",
+ *  {
+ *    errors: { 404: false }
+ *  }
+ * )
  * @param {string} path - path to resource
  * @param {RequestOptions} [options={}] - An object containing query params and headers for the request
  * @returns {operation}
```

Comment:
I think we need to documentation for final state for each `http.namespace` function 

```
/**
 * State object
 * @typedef {Object} HttpState
 * @property data - the parsed response body
 * @property response - the response from the HTTP server, including headers, statusCode, body, etc
 * @property references - an array of all previous data objects used in the Job
 **/
```

Reply:
This is done in openMRS v5 (coming soon!). We won't duplicate the work here.

Reply:
Got you 👍🏽